project/proxy/proxy_server_api.py

```diff
@@ -39,12 +48,14 @@ def push_command(self, command: str):
             return self.fail_response_no_server
 
         try:
-            server_response = self.server_api_list[last_index_call].push_command(command_data)
+            server_proxy_last_index = Pyro4.Proxy(self.server_api_list[last_index_call])
```

Comment:
don't use postfix _index for non integer

Reply:
ok will do